src/components/project-list.ts

```diff
@@ -1,74 +1,97 @@
 //Project ListClass
-///<reference path="base-components.ts"/>
-///<reference path="../models/drag-and-drop.ts"/>
-///<reference path="../decorators/autobind.ts"/>
-///<reference path="../state/project-state.ts"/>
-///<reference path="../models/project.ts"/>
-namespace App {
-  export class ProjectList
-    extends Component<HTMLDivElement, HTMLElement>
-    implements DragTarget
-  {
-    assignedProjects: Project[] = [];
-    constructor(private type: "active" | "finished") {
-      super("project-list", "app", false, `${type}-projects`);
-
-      this.configure();
-      this.renderContent();
-    }
+import { Component } from "./base-components.js";
+import { DragTarget } from "../models/drag-and-drop.js";
+import { Project } from "../models/project.js";
+import { autobind } from "../decorators/autobind.js";
+import { ProjectItem } from "./project-item.js";
+import { ProjectStatus } from "../models/project.js";
+import { projectState } from "../state/project-state.js";
+export class ProjectList
+  extends Component<HTMLDivElement, HTMLElement>
+  implements DragTarget
+{
+  assignedProjects: Project[] = [];
+  constructor(private type: "active" | "finished" | "to-do" | "review") {
```

Comment:
We can have a type defined with all 4 options, and use a variable of that type anywhere we reference the project state.